spec/directives/use/error/with.hrx

```diff
@@ -351,3 +351,109 @@ Error: This variable was not declared with !default in the @used module.
   |                   ^^^^^
   '
   input.scss 1:19  root stylesheet
+
+<===>
+================================================================================
+<===> unused_distributed_vars/README.md
+A module may load different sets of variables across multiple files triggering
+validation of re-using the same module configuration multiple times.
+
+These tests verify the configuration is reused by opaque ID across multiple
+source files preventing unexpected behaviors when variables are declared on
+separate files with validations that occur when a module is used more than once.
+
+
+<===>
+================================================================================
+<===> unused_distributed_vars/single_use/input.scss
+@use 'module' with (
+  $foo: 'foo',
```

Comment:
Nit: [Use single-letter names for irrelevant placeholders](https://github.com/sass/sass-spec/blob/main/STYLE_GUIDE.md#do-use-single-letter-names-for-irrelevant-placeholders).

Reply:
done